---
title: "Differential expression analysis with Bulk RNA-seq"
format: html
---

This workshop covers the basics of differential expression analysis at the gene level with bulk RNA-seq data, *assuming a reference genome and both structural and functional annotation are available*. This workflow can apply to model organisms, and others with published genomic resources. 

See [this site](https://bioinformatics.uconn.edu/cbc-workshops/) for our workshop schedule and registration link. 

::: {.callout-note}
For species where genomic resources are not available, we offer [a version of the workshop](bulkRNAdeNovo.qmd) that includes *de novo* transcriptome assembly and functional annotation. If a genome is available and neither structural nor functional annotation is available, we offer an independent workshop entirely on [genome annotation](genomeAnnotation.qmd)
:::

### Schedule

- **Prior to workshop**
  - Prior to the synchronous portion of the workshop, attendees complete a [self-guided introduction to our high performance computing cluster](introHPC.qmd) where they will learn to connect, work at the command line using the BASH shell on a Linux operating system, and submit work using the HPC job scheduler SLURM. 
- **Day 1**
  - Intro to sequencing technologies
  - Intro to RNA-seq data
  - Study design considerations
  - Retrieving data from NCBI
  - QC and Trimming
- **Day 2**
  - Reference alignment
  - Expression quantification
- **Day 3**
  - Expression quantification continued.
  - Differential expression analysis
  - Functional annotations
  - Gene set enrichment and over-representation analysis
  - Visualizing RNA-seq data and differential expression results


### Data

As an example dataset, we use RNA-seq data from:

>Reid, N. M., Proestou, D. A., Clark, B. W., Warren, W. C., Colbourne, J. K., Shaw, J. R., ... & Whitehead, A. (2016). The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish. Science, 354(6317), 1305-1308.