---
title: "Workshops Overview"
format: html
---

The CBC offers a monthly training workshop series that covers commonly applied workflows in bioinformatics. These workshops are typically virtual and conducted over Microsoft Teams, though if there is demand we can schedule an in-person workshop. 

Workshops are conducted live, with opportunities to ask questions of instructors before, during and after each session. 

These workshops are highly practical. We aim to reduce the barriers researchers face in learning a new type of analysis, and are especially useful to those who are in the planning stages for a new project, or who have data they need to begin analyzing. 

Virtual workshops are open to anyone, inside or outside of UConn, though external participants are charged a higher rate. **See [our schedule page](https://bioinformatics.uconn.edu/cbc-workshops/) for the current schedule and rates and instructions for how to register**. See the left-hand navigation bar for details of individual workshops we offer.

### Key features of our workshop series

Our approach in the workshops is to demonstrate a fully worked example of an analysis of a real dataset, providing conceptual background alongside a detailed explanation of the code to run the software required. 

Some key features of these workshops:

- **Short:** Core content covering the workflow is delivered over three half-days (10am-2pm Eastern time). 
- **Real data**: We walk through all the steps required to analyze a real, typically publicly available, data set, highlighting QC problems and solutions along the way. 
- **HPC training** 
  - We assume no prior experience with linux or high-performance computing. 
  - Before the core content begins you will do a self-guided introduction. 
  - Use real high-performance computing infrastructure during the workshop. 
  - If you are a UConn affiliate, you will gain the knowledge to begin independent analysis on the Mantis cluster. 
  - Workshop HPC accounts remain active for two weeks, giving participants the chance to further explore the material. 
- **Open code**: All code required to reproduce the analysis provided in a public Git repository. This repository can serve as a useful template for future analyses. 
- **Session recordings**: Recordings of each day's sessions are distributed to participants. 

### Learning Goals

- Students will be able to describe the components of the focal workflow (e.g. differential expression with RNA-seq) including:
  - Software required.
  - Input and output data.
  - Common data formats. 
  - Typical quality control steps.
  - Commonly used statistical approaches.
- Students will be able to describe common study design considerations. 
- Students will be able to interpret the code provided. 
- Students will be able to execute the code provided on UConn's HPC resources (Xanadu or Mantis). 

#### Limitations

The short format of these workshops imposes some limitations. 
- Many workshops are "cooking show" style. For certain types of analyses (e.g. genome assembly), we provide precomputed results to speed the workshop along. However, students will follow along on the HPC, exploring input and results, and will have example code they can run from scratch or modify during the two week period workshop accounts are active.
- In the context of [Bloom's Taxonomy](https://stearnscenter.gmu.edu/knowledge-center/course-and-curriculum-redesign/blooms-taxonomy/) we aim to get you through the "remembering" and "understanding" levels and into "application". But bioinformatics is hard, and learning more deeply will take time. 
